hydra-main/src/main/java/com/addthis/hydra/job/mq/HostState.java

```diff
@@ -391,6 +391,24 @@ public void setMax(HostCapacity max) {
         this.max = max;
     }
 
+    public long getAvailDiskBytes() {
+        if ((max == null) || (used == null)) {
+            return 1; // Fix some tests
+        }
+        return max.getDisk() - used.getDisk();
+    }
+
+    public double getDiskUsedPercent() {
+        return getDiskUsedPercentModified(0);
+    }
+
+    public double getDiskUsedPercentModified(long reduce) {
+        if ((max == null) || (used == null) || (max.getDisk() <= 0)) {
+            return 0;
+        }
+        return (double) (used.getDisk() / (max.getDisk() - reduce));
```

Comment:
Should check return value to make sure it's in the range of [0, 1]. Also add javadoc here and/or comment where it is called to explain usage/reasoning